apps/nextjs-website/src/config.ts

```diff
@@ -39,7 +39,7 @@ export const baseUrl = isProduction
 
 export const defaultOgTagImage = `${baseUrl}/images/dev-portal-home.jpg`;
 export const resetResendEmailAfterMs = 4_000;
-export const fetchWebinarsQuestionsIntervalMs = 250;
+export const fetchWebinarsQuestionsIntervalMs = 2_500;
```

Comment:
I'm not sure that this interval should return to the previous value.
I think that - during a live - 2.5 seconds plus the response time is too much and 1 second could be good compromise.
@datalek what do you think about this?

Reply:
I don't think the difference is too relevant. So choose the one you think is most appropriate 🙂